src/app/components/work-item-detail/work-item-detail.component.ts

```diff
@@ -850,7 +770,14 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
           }
         }
       });
-      this.save(payload);
```

Comment:
We need to emit the emitEditWI event in the assignUser function as save function is no longer emitting it.